Remotion/Web/Core/Res/Themes/NovaGray/HTML/StyleUtility.js

```diff
@@ -26,31 +26,31 @@ StyleUtility.CreateBorderSpans = function (selector)
 
 StyleUtility.AddBrowserSwitch = function ()
 {
-  var browser;
-  if (navigator.appVersion.indexOf ('Edge') !== -1)
+  function searchBrowser(brands, search)
   {
-    browser = 'edge';
+    return brands.some(c => c.brand === search);
   }
-  else if (navigator.appVersion.indexOf ('Chrome') !== -1)
-  {
-    browser = 'blink';
-  }
-  else if (navigator.appVersion.indexOf ('OPR') !== -1)
-  {
-    browser = 'blink';
-  }
-  else if (navigator.appVersion.indexOf('WebKit') !== -1)
-  {
-    browser = 'webkit';
-  }
-  else if (navigator.appName === 'Netscape')
+
+  let browser;
+  if (navigator.userAgentData)
   {
-    browser = 'mozilla';
+    const brands = navigator.userAgentData.brands;
+    if (searchBrowser(brands, 'Microsoft Edge') || searchBrowser(brands, 'Google Chrome') || searchBrowser(brands, 'Opera'))
+      browser = 'blink';
   }
-  else
+
+  if(!browser)
   {
-    browser = 'browserUnknown';
+    if ((navigator.userAgent.indexOf('Edg/') !== -1) || (navigator.userAgent.indexOf('Chrome') !== -1) || (navigator.userAgent.indexOf('OPR') !== -1))
```

Comment:
same as with line 38, please split the IF.

thought: you could declare all the browser values as const-vars if you like.